res/TensorFlowPythonExamples/examples/scatter_nd/__init__.py

```diff
@@ -0,0 +1,10 @@
+# This example is a modified example from TF API documentation
+
+import tensorflow as tf
+
+indices = tf.compat.v1.constant([[0], [2]])
+updates = tf.compat.v1.constant([[[5, 5, 5, 5], [6, 6, 6, 6], [7, 7, 7, 7], [8, 8, 8, 8]],
+                                 [[5, 5, 5, 5], [6, 6, 6, 6], [7, 7, 7, 7], [8, 8, 8,
+                                                                             8]]])
```

Comment:
Maybe, space isn't intended?

Reply:
Unfortunately that's how the format checker wants it